src/sites/link/adf.ly.js

```diff
@@ -32,6 +32,19 @@
     },
   });
 
+  _.register({
+    rule: {
+      query: /lp=adfly_allow&.*href=([^&]+)/,
+    },
+    async start (m) {
+      const url = decodeURIComponent(m.query[1]);
```

Comment:
I just copied this method from line 20 above

the other method sets some cookies, though, not sure why

Reply:
It was needed for reset locked-page-cookie.
reference: 3cd6712b92fd0567226a82862822dc9f20bf652a